pkg/service/backup/restore_worker_index.go

```diff
@@ -194,11 +194,27 @@ func (w *indexWorker) workFunc(ctx context.Context, run *RestoreRun, target Rest
 
 			pr.setRestoreCompletedAt()
 			w.insertRunProgress(ctx, pr)
-
 			restoredBytes := pr.Downloaded + pr.Skipped + pr.VersionedProgress
 			w.metrics.DecreaseRemainingBytes(w.ClusterID, target.SnapshotTag, w.location, w.miwc.DC, w.miwc.NodeID,
 				pr.Keyspace, pr.Table, restoredBytes)
 
+			// Calculate restore progress in percents
+			var progress float64
+			if fm.Size == 0 {
+				progress = 100
+			} else {
+				progress = float64(restoredBytes) / float64(fm.Size) * 100
+			}
```

Comment:
@charconstpointer this formula won't work.
`restoredBytes` is just a chunk of restored keyspace/table from backup location.

You must remember the progress value for current index.
To do it, please just add `progress` var to `indexWorker` struct, then 
`w.progress += float64(restoredBytes) / float64(fm.Size) * 100`

Reply:
now that I think about it, `w.progress` will exceed 100% do we have to divide it by `len(w.hosts)`?

Reply:
Applied the propsed changes, now it reports 0-100 per unique metric label set
<img width="1715" alt="Screenshot 2023-06-28 at 13 55 56" src="https://github.com/scylladb/scylla-manager/assets/35968924/0ec85ff2-fbcc-42b4-b5e1-f2a9c3f377a5">

Reply:
As per our slack conversation.
Having access to the size of the data to restore + `restored_bytes` var you are able to track the progress.
Accumulated progress must be shared between all hosts that participate in the restore.

Reply:
Seems like the workaround we've discussed works fine
<img width="1707" alt="Screenshot 2023-07-03 at 11 33 20" src="https://github.com/scylladb/scylla-manager/assets/35968924/582c6044-527d-4c27-8fc8-65a13557ca6b">